views/pages/ViewChartPage/index.jsx

```diff
@@ -134,7 +148,13 @@ const ViewChartPage = () => {
         </span>
       </div>
       <ChartFilterForm initialValues={graph.filters} onSubmit={onSubmit} />
-      <BarGraph graph={graph} />
+      <BarGraph
+        graph={graph}
+        tooltipPosition={tooltipPosition}
+        handleTooltipPosition={handleTooltipPosition}
+        displayLeft={displayLeft}
+      />
+      <div style={{ height: '60px' }} />
```

Comment:
Why are we using an empty div here? Can we do this with `padding`?

Reply:
so would margin work instead?

Reply:
Maybe. I think it's worth a shot but Recharts is pretty weird about style customization. Did some pairing with Iván that gave me some perspective on that 😓